src/test/java/no/difi/sdp/client2/smoke/SmokeTest.java

```diff
@@ -89,15 +88,15 @@ private static Noekkelpar avsenderNoekkelpar() {
         return Noekkelpar.fraKeyStoreUtenTrustStore(keyStore, virksomhetssertifikatAliasValue, virksomhetssertifikatPasswordValue);
     }
 
-    private static Organisasjonsnummer getOrganisasjonsnummerFraSertifikat() {
+    private static String getOrganisasjonsnummerFraSertifikat() {
```

Comment:
Hvorfor denne endringen? Er det ikke _egentlig_ ryddigst at denne metoden returnerer et `Organisasjonsnummer`? 

Regner med den skal dyttes inn i factory-metoden til `AktørOrganisasjonsnummer` på et eller annet tidspunkt – men der kan du jo ha en overload som tar inn et `Organisasjonsnummer` i tillegg til `String`, for eksempel. Litt av poenget her er vel å øke typesikkerheten ved å bli kvitt `String`s som sendes rundt om kring :)